server/controllers/stock/core.js

```diff
@@ -259,6 +260,22 @@ async function getLotsDepot(depotUuid, params, finalClause) {
     emptyLotToken = 'HAVING quantity = 0';
   }
 
+  // Add extra fields for assignments if the query is for assets
+  let assignmentFields = '';
+  let assignmentJoin = '';
+  if ('is_asset' in params && params.is_asset === '1') {
```

Comment:
I am almost a fan of adding a separate route for this.  Given how much performance is hampered by these routes. 

What do you think?  Is it worth adding a separate `stock/assets` route for this?

Reply:
I'm okay with that, but I'd prefer to do it in a separate issue/PR.

Reply:
Understood.  Will make one and link it here.